### DISCUSSION TITLE
so-firewall traceback

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#8259] [General] [answered]
Discovered a traceback from 'so-firewall --defaultports listportgroups' after attempting an upgrade from 2.3.90 to 2.3.130 which went south in our distributed, airgapped, environment. Wondering if this is expected behavior.

**Comment by dougburks** [comment, marked as answer]
This should be resolved in the upcoming 2.3.140 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/8264
